### ISSUE TITLE
Spinner hidden text not working as documented

### ISSUE THREAD

**Issue by Kritzefitz** [isosphere/yew-bootstrap#20]
The documentation of `yew_bootstrap::component::Spinner` says that visually hidden text in the spinner should be added as follows:

```
<Spinner>
  {"hidden text"}
</Spinner>
```

However, that example renders like this:

![image](https://github.com/isosphere/yew-bootstrap/assets/8547449/d97aa7f3-deb3-4bb9-a2ec-f7e466564ab0)

Looking at the source of the `Spinner` component

```
        html! {
            <div class={classes} role="status">
                <span class="visually-hidden">
                    {&props.text}
                </span>
                { for props.children.iter() }
            </div>
        }
```

it looks like the `text` attribute is actually rendered where the hidden text would be supposed to be rendered. But since its type is `bool` it can't really be used to pass any useful state either.

**Comment by isosphere** [comment]
Thank you for your report @Kritzefitz , I can confirm your findings.

Please check out #21 for my proposed solution and let me know if it meets your needs.

**Comment by isosphere** [comment]
Closed as completed! Fixed in 0.5.16.
